### ISSUE TITLE
return breakpoints to _root scss

### ISSUE THREAD

**Issue by patrickatwsrn** [twbs/bootstrap#36081]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

I know this has been a thing in earlier versions:

scss
  @each $bp, $value in $grid-breakpoints {
    --breakpoint-#{$bp}: #{$value};
  }

<img width="280" alt="image" src="https://user-images.githubusercontent.com/68318893/160614348-209c0962-7200-4843-b560-73274c94f617.png">
https://github.com/twbs/bootstrap/blob/main/scss/_root.scss

but it has been removed from _root.scss with pull request 29020: https://github.com/twbs/bootstrap/pull/29020
https://github.com/twbs/bootstrap/commit/cdb6504e4803538e28a251a7f9634df2d580cd3e

### Motivation and context

The reasoning for removing this part was: 

> "Media queries do not support CSS custom properties (CSS variables), so we've had some useless variables lying around since shipping v4. This finally removes them and the associated documentation content that explained the situation."

There was not much of a discussion, but @mdo rightfully pointed out "No, because it'd be a breaking change if someone were using the variables elsewhere."

Though @MartijnCuppens gave this point a thumbs up, he later approved the PR (I'd really like to hear why he changed his mind). 

My 5 cents on this topic:

1. it was a very useful tool to understand the breakpoints working in a bootstrap project. Just one look and you knew what to expect.
2. yes media queries do not support CSS custom properties, but that doesn't mean that you can't make use of this variable in another context
3. You can grab this settings with javascript and you can do a lot of useful stuff with it.

Sidenote: I'm aware that they are not very helpful if somebody overwrites the default breakpoints with custom ones, but this is less of a problem if you are working with your own code.

So, please bring them back to the _root

Regards,

Patrick

**Comment by patrickatwsrn** [comment]
I also remember that the breakpoint where the mobile navigation kicks in was once listed inside root, but I can't find proof for this. It would be beneficial to add this one too.

**Comment by shaack** [comment]
I need these variables to get my breakpoint detection script working: https://github.com/shaack/bootstrap-detect-breakpoint. Otherwise I have no idea how to detect the current breakpoint from JS.

I also created a duplicate ticket for this: https://github.com/twbs/bootstrap/issues/36094 (will close it soon)

Although this should be present according to [the documentation](https://getbootstrap.com/docs/5.1/customize/css-variables/#grid-breakpoints): "[...]we include our grid breakpoints as CSS variables (except for xs) [...]"

I also created a pull request to get it working again: https://github.com/twbs/bootstrap/pull/36095

These variables are very important for breakpoint detection in JavaScript, so I'm in favor of adding them back into the code.
